pyLDAvis/js/ldavis.v3.0.0.js

```diff
@@ -512,7 +517,7 @@ var LDAvis = function(to_select, data_or_file_name, color1, color2) {
             .attr("x", 0)
             .attr("y", mdsheight + 10 + (8/2)*barguide.height + 5)
             .style("dominant-baseline", "middle")
-            .text("2. relevance(term w | topic t) = \u03BB * p(w | t) + (1 - \u03BB) * p(w | t)/p(w); see Sievert & Shirley (2014)");
+            .text("2. relevance(term w | topic t; \u03BB) = \u03BB * log(p(w | t)) + (1 - \u03BB) * log(p(w | t)/p(w)); see Sievert & Shirley (2014)");
```

Comment:
https://github.com/bmabey/pyLDAvis/issues/261#issuecomment-2062986867

You don't mention this formula change?

Reply:
I changed it at line 511.  I am not sure what you mean.